client/state/reader/streams/utils.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { last } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { sameXPost } from 'reader/stream/utils';
+
+/**
+ * Add the post URL from a duplicate x-post to an existing post key
+ *
+ * @param {object} postKey1 First post key
+ * @param {object} postKey2 Second (duplicate) post key
+ * @return {array} Array of post key objects
```

Comment:
Doesn't this return an Object instead of an Array?

Reply:
You're right! It returns a post key object. Good spot - I'll fix.